File: baekjoon_all/10000+/boj_10101.py
# Solve 2023-03-14

import sys

input = lambda : sys.stdin.readline().rstrip()

angles = [int(input()) for _ in range(3)]

if sum(angles) != 180:
    print("Error")
elif angles[0] == 60 and angles[1] == 60:
    print("Equilateral")
elif angles[0] == angles[1] or angles[1] == angles[2] or angles[2] == angles[0]:
    print("Isosceles")
else:
    print("Scalene")

File: baekjoon_all/10000+/boj_10101.cpp
// Solve 2022-07-12
// Update 2023-03-14

#include <bits/stdc++.h>
using namespace std;

#ifdef BOJ
#define BOJTEST(x) ((void)0)
#else
#define BOJTEST(x) cout << "[Debug] " << #x << ':' << x << '\n'
#endif
#define FASTIO ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL); // boj_15552.cpp
#define SETPRECISION(n) cout << fixed;cout.precision(n); // boj_1008.cpp
#define SIZE(v) (int)v.size()
#define ALL(v) v.begin(),v.end()
using ll = long long;
using uint = unsigned int;
using ull = unsigned long long;

int main() {
    FASTIO;

    int a, b, c;
    cin >> a >> b >> c;

    if (a + b + c != 180) {
        cout << "Error\n";
    }
    else if (a == 60 && b == 60) {
        cout << "Equilateral\n";
    }
    else if (a == b || b == c || c == a) {
        cout << "Isosceles\n";
    }
    else {
        cout << "Scalene\n";
    }

    return 0;
}
